emails/build-emails

```diff
@@ -28,17 +36,41 @@ function renderTemplate(createTemplate, name) {
     templateText = handlebars.compile(templateText)(mockData)
   }
 
-  fs.mkdirSync(DIST_DIR, { recursive: true })
-  fs.writeFileSync(path.join(DIST_DIR, name + '.html'), template, 'utf8')
-  fs.writeFileSync(path.join(DIST_DIR, name + '.txt'), templateText, 'utf8')
+  return {subject, name, template, templateText}
+}
+
+async function syncPostmark(files) {
+  if (!POSTMARK_SERVER_API_TOKEN) {
+    console.error('You must specify a valid POSTMARK_SERVER_API_TOKEN environment variable.')
+    process.exit(1)
```

Comment:
Rather than exit here, should we just throw an error?

Reply:
changed!